### ISSUE TITLE
Pustulosis

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6366]
**Preferred term label:**
figure out hierarchy
see https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7508911/

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://pubmed.ncbi.nlm.nih.gov/19494218/  8/8
https://omim.org/entry/612852

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Probably better to call the term Pustular rash

https://bestpractice.bmj.com/topics/en-us/776

6/6 https://pubmed.ncbi.nlm.nih.gov/25029335/
https://omim.org/entry/615934

**Comment by pnrobinson** [comment]
A rash composed of pustular lesions. A pustule is defined as a vesicle or bulla containing purulent material. It varies in size and may occur at different levels within the epidermis: subcorneal, intraepidermal, or basement membrane zones. The pustules may or may not be sterile and are normally filled with neutrophils.  PMID:25805031

**Comment by pnrobinson** [comment]
added
